README.md

```diff
@@ -832,6 +832,7 @@ _Games, game servers and control panels._
 - [Mindustry](https://mindustrygame.github.io/) - Factorio-like tower defense game. Build production chains to gather more resources, and build complex facilities. ([Source Code](https://github.com/Anuken/Mindustry)) `GPL-3.0` `Java`
 - [Minetest](https://www.minetest.net/) - An open source voxel game engine. Play one of our many games, mod a game to your liking, make your own game, or play on a multiplayer server. ([Source Code](https://github.com/minetest/minetest)) `LGPL-2.1/CC-BY-SA-3.0/Other` `C++`
 - [Net64+](https://net64-mod.github.io) `⚠` - Net64 aka SM64O allows playing Super Mario 64 in an online multiplayer mode. Net64+ is the official continuation of the program and features an integrated server list. ([Source Code](https://github.com/Tarnadas/net64plus-server/), [Clients](https://github.com/Tarnadas/net64plus/)) `MIT` `Nodejs/Typescript`
+- [node-virtual-gamepads](https://github.com/jehervy/node-virtual-gamepads) - Turn your smartphone into a game controller, keyboard, or touchpad for a remote Linux OS machine. `MIT` `JavaScript/CoffeScript`
```

Comment:
Platform/language is `Nodejs`

`sudo node main.js` -> why does it need root?

Reply:
In fact, this discussion came up before: https://github.com/jehervy/node-virtual-gamepads/issues/57
While it is technically possible to run this a non-root user, you'd have to a) change the default port from 80 to a not well-known port and b) (the harder part) make sure the user executing the server has permission to write to `/dev/uinput`. If you insist, I'm willing to add a section to the readme explaining how to do that (at least for Ubuntu-like systems), but I wouldn't want to make it the recommended way. It will cause more people to need assistance in setting up the app.

Reply:
So?

Reply:
If you can fix the `JavaScript/CoffeeScript` > `Nodejs` thing I will merge it.

About permissions level, yes it would probably be better if the docs explained how to configure them in a proper and safe way (`sudo groupadd uinput && sudo chgrp uinput /dev/uinput && sudo chmod g+rw /dev/uinput`, and run your app under the `uinput` user. Untested.), at least add a note explaining that `root` access is a simpler but unsafe way to get the required permissions.

Reply:
I already changed `JavaScript/CoffeeScrip` to `Nodejs/CoffeScript`. Actually there also is some JavaScript involved in the front end… But anyway.

I will change the docs soon. If this PR is still open till then I will ping you again, when I'm done. Thanks for maintaining this awesome list.